packages/core/src/i18n/translations.ts

```diff
@@ -0,0 +1,312 @@
+    'Sign In with Google': 'Se connecter avec Google',
+    'Sign In with AWS': 'Se connecter avec AWS',
+    'Sign Up': "S'inscrire",
+    'Sign Out': 'Déconnexion',
+    'Forgot Password': 'Mot de passe oublié',
+    Username: "Nom d'utilisateur",
+    Password: 'Mot de passe',
+    'Change Password': 'Modifier le mot de passe',
+    Change: 'Modifier',
+    'New Password': 'Nouveau mot de passe',
+    Email: 'Email',
+    'Phone Number': 'Numéro de téléphone',
+    'Confirm a Code': 'Confirmer un code',
+    'Confirm Sign In': 'Confirmer la connexion',
+    'Confirm Sign Up': "Confirmer l'inscription",
+    'Back to Sign In': 'Retour à la connexion',
+    Send: 'Envoyer',
+    'Send Code': "M'envoyer un code",
+    Confirm: 'Confirmer',
+    SMS: 'SMS',
+    'Confirm SMS Code': 'Confirmer le code SMS',
+    'Confirm TOTP Code': 'Confirmer le code TOTP',
+    'Resend a Code': 'Renvoyer un code',
+    Submit: 'Soumettre',
+    Skip: 'Passer',
+    Verify: 'Vérifier',
+    'Verify Contact': 'Vérifier le contact',
+    Code: 'Code',
+    'Confirmation Code': 'Code de confirmation',
+    'Account recovery requires verified contact information':
+      'La récupération du compte nécessite des informations de contact vérifiées',
+
+    'User does not exist': "L'utilisateur n'existe pas",
+    'User already exists': "L'utilisateur existe déjà",
+    'Incorrect username or password': 'Identifiant ou mot de passe incorrect',
+    'Invalid password format': 'Format de mot de passe invalide',
+    'Invalid phone number format': `Format de numéro de téléphone invalide. Veuillez utiliser un format +12345678900`,
+    'Username/client id combination not found.': "L'utilisateur n'existe pas",
+    'Network error': 'Erreur réseau',
+    'Sign in to your account': 'Connexion à votre compte',
+    'Forgot your password? ': 'Mot de passe oublié ? ',
+    'Reset password': 'Réinitialisez votre mot de passe',
+    'No account? ': 'Pas de compte ? ',
+    'Create account': 'Créer un compte',
+    'Create Account': 'Créer un compte',
+    'Have an account? ': 'Déjà un compte ? ',
+    'Sign in': 'Se connecter',
+    'Create a new account': 'Créer un nouveau compte',
+    'Reset your password': 'Réinitialisez votre mot de passe',
+    'Enter your username': "Saisissez votre nom d'utilisateur",
+    'Enter your password': 'Saisissez votre mot de passe',
+    'Enter your phone number': 'Saisissez votre numéro de téléphone',
+    'Enter your email': 'Saisissez votre adresse email',
+    'Enter your code': 'Saisissez cotre code de confirmation',
+    'Lost your code? ': 'Vous avez perdu votre code ? ',
+    'Resend Code': 'Renvoyer le code',
+    or: 'ou',
+    'An account with the given email already exists.':
+      'Un utilisateur avec cette adresse email existe déjà.',
+    'Username cannot be empty': "Le nom d'utilisateur doit être renseigné",
+  },
+
+  es: {
+    'Sign in to your account': 'Iniciar sesíon',
```

Comment:
We can add in typing...but I wonder if "forcing" parity between the languages would work. We are going to rely on the open source community for translation support, and I'm picturing a scenario when a contributor will want to add a new string, or only a couple strings of translations. In that case, forcing parity between all languages might almost act like a blocker. Maybe I'm thinking about it wrong.

Reply:
Ah, that's a fair point. I agree, let's keep it as is now. Maybe we can do Partial<> just so that we have vscode autocomplete, but this is not an issue now and I don't want it to be a blocker.